ulib/FStar.Tactics.PatternMatching.fst

```diff
@@ -166,21 +166,18 @@ type varname = string
 type qn = string
 
 type pattern =
-| PAny: pattern
```

Comment:
Oh, oops ... this is an unrelated change in my branch. 

It removes a problematic exploitable assume in this code.